examples/mermaid/comprehensive_cardinalities_and_types_w_props.mm.txt

```diff
@@ -0,0 +1,33 @@
+erDiagram
```

Comment:
Maybe group relations at the top. Then add classes with content below.

Reply:
Do you mean [as in the mermaid examples](https://mermaid.js.org/syntax/entityRelationshipDiagram.html#attributes) (per class) or do you mean globally all relations first, then all classes? I'm not sure what would be the advantage, unless I'm missing any would prefer to keep the current styl.e

Reply:
I meant first putting all these lines:

```
Customer ||--o{ Address : "Customer.addresses"
Customer ||--o{ Order : "Customer.orders"
...
```

And then all the class definitions such as:
```
Customer {
string name "req"
object attributes "req"
boolean company "req"
}
```

But this is really personal taste and doesn't really matter that much.

I see that in the example you provided they interleave the two. So maybe this is the common pratice.